front-end/cypress/e2e-smoke/pages/pageUtils.ts

```
@@ -123,7 +123,10 @@ export class PageUtils {
 
   static clickButton(name: string, alias = '', force = false) {
     alias = PageUtils.getAlias(alias);
-    cy.get(alias).contains('button', name).as('btn');
+    cy.get(alias)
+      .contains('button', name)
+      .first()
```

concerned about if this will silence issues that might fail loudly.  like i would want to know that i wrote my test ambiguously so i could add specificity rather than it picking the first one and me not knowing whether that's the element i meant for it to pick